src/server/pachyderm_test.go

```diff
@@ -4910,7 +4921,9 @@ func TestDatumStatusRestart(t *testing.T) {
 }
 
 func TestUseMultipleWorkers(t *testing.T) {
-	t.Skip("flaky")
```

Comment:
Same comment as below, I think we may need to leave this in for now

Reply:
same as above. Will check in master and put it back if it's definitely there too.